KlingonAssistant/data/mem-09-n.xml

```diff
@@ -2557,7 +2557,7 @@ See {SIq:v} for an explanation of the usage of these finger verbs.</column>
       <column name="entry_name">noq</column>
       <column name="part_of_speech">n</column>
       <column name="definition">nipple</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">Nippel, Brustwarze, Trinksauger (für Flaschen)</column>
```

Comment:
Per your request I've split this into two entries: https://github.com/De7vID/klingon-assistant/pull/237